download_filing_docs.R

```diff
@@ -0,0 +1,41 @@
+
+raw_directory <- Sys.getenv("EDGAR_DIR")
+
+library(RPostgreSQL)
+library(dplyr)
+
+pg <- dbConnect(PostgreSQL())
+
+all_files<-tbl(pg, sql("select * from edgar.filing_docs limit 140"))
+files_downloaded<-tbl(pg, sql("select * from edgar.filing_docs_processed limit 120"))
+
+if(exists("files_downloaded")){files<-all_files %>% anti_join(files_downloaded %>% filter(downloaded == TRUE)) %>% collect() %>% mutate(html_link = file.path(gsub("-","",gsub(".txt","",file_name)),document)) } else {files<-all_files %>% collect() %>% mutate(html_link = file.path(gsub("-","",gsub(".txt","",file_name)),document)) }
```

Comment:
Very ugly code here.

Reply:
Note that cases where `downloaded == FALSE` are presumably cases where the download failed. Not clear that we want to keep trying these cases. Better to `anti_join` those cases too and solve somewhere else.